app/code/core/Mage/Core/Model/Email/Queue.php

```diff
@@ -238,13 +238,13 @@ public function send()
 
                 try {
                     $mailer->send();
+                    unset($mailer);
```

Comment:
Why move the unset?

Reply:
I think the `unset ` comes from line 245 https://github.com/OpenMage/magento-lts/pull/1109/files#diff-e408845a99067b7da8df6e1c971d74dd81e8d02f15b4b6111ce1d4389ae45e49L245